cfme/services/myservice/ssui.py

```diff
@@ -47,14 +49,16 @@ def is_displayed(self):
 
     notification = Notification()
     policy = SSUIDropdown('Policy')
-    power_operations = SSUIDropdown('Power Operations')
+    power_operations = VersionPick(
+        {Version.lowest(): SSUIDropdown("Power Operations"), "5.10": Kebab()}
```

Comment:
Use VersionPicker instead

Reply:
@sshveta can you please explain why?

Reply:
@digitronik : Actually there was a PR by Dmitry sometime back in which he wrapped 
cfme.utils.version.pick to  a method of VersionPicker and removed versionpick from most of the places . I have been using versionpicker since then .
https://github.com/ManageIQ/integration_tests/pull/7543 and 
https://github.com/ManageIQ/integration_tests/pull/7600

Reply:
But its completely `optional`  as its wrapped. In my condition; I am not using an extended feature of `ViersionPicker` so its valid as per `WT` guideline and current implementation; 

As per my observation and understanding with current `collection` implementation no need of `VersionPicker`.

Cases where  we need to take  action as per `version`:
1. In `views class`:
where we can not access `appliance` object at the initial stage (`l.e.` `locator`, `Widget` selection) then go with `VersionPick`.
2. In `views methods` or `properties`:
where possible to access `appliance` object directly free to go with a condition.
3. In `Navigation`:
If we need to return `view` as per `appliance version` then we can go with `VIEW property` means
```
@property
def VIEW(self):
    return view1 if self.appliance.version < 'x.y' else view2
```

I think it's better to make one guideline; where we should accept `VersionPick`, `VersionPicker` and `appliance.version` condition. adding `tech-debt` tag.

Reply:
@psav @quarckster @mshriver @izapolsk @RonnyPfannschmidt Please have a look... It will help to avoid confusion...

Reply:
@digitronik The intent of creating `VersionPicker` to unify different version picker implementations. I would insist to use `VersionPicker`.

Reply:
I'd add that VersionPicker is prefered for widgetastic stuff because it can be correctly resolved by wt methods and etc